tests/test.program.js

```diff
@@ -202,17 +249,38 @@ describe('program.main', () => {
       });
   });
 
+  it('can get the program version', () => {
+    const fakeVersionGetter = sinon.spy(() => '<version>');
+    const fakeCommands = fake(commands, {
+      build: () => Promise.resolve(),
+    });
+    const projectRoot = '/pretend/project/root';
+    // For some reason, executing --version like this
+    // requires a command. In the real CLI, it does not.
+    return run(['--version', 'build'],
```

Comment:
Hmm, I actually cannot reproduce this with an isolated yargs program. It's probably not worth spending too much time on it. Here's what I tried to mimic what web-ext does:
````javascript
var yargs = require('yargs');
yargs
  .help('help')
  .demand(1)
  .strict()
  .exitProcess(false)
  .version(function() {
    return "the version!"
  })
  .command('thing', '', function(yargs) {
  }, function() {
    console.log('running thing');
  })
  .argv;
````